spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindBadEndOfStreamCheck.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * FindBugs - Find bugs in Java programs
```

Comment:
This should be "SpotBugs" since it never existed in FindBugs, and the University of Maryland has no claim on it.